tests/kuttl/delivery/deliverable-status/02-assert.yaml

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 VMware
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: example-second-configmap
+data:
+  player_initial_lives: "5"
+  player_loves: "knitting"
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: example-config-configmap
+data:
+  favorite_color: "5"
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: example-config
+data:
+  best_name: "5"
+
+---
+apiVersion: carto.run/v1alpha1
+kind: Deliverable
+metadata:
+  name: petclinic
+status:
+  conditions:
+    - type: DeliveryReady
+      status: "True"
+      reason: Ready
+    - type: ResourcesSubmitted
+      status: "True"
+      reason: ResourceSubmissionComplete
+    - type: Ready
+      status: "True"
+      reason: Ready
+  deliveryRef:
+    name: responsible-ops---deliverable-status
+    kind: ClusterDelivery
+  resources:
+    - name: source-provider-first
+      stampedRef:
+        apiVersion: v1
+        kind: ConfigMap
+        name: example-first-configmap
+      templateRef:
+        apiVersion: carto.run/v1alpha1
+        kind: ClusterSourceTemplate
+        name: example-first---deliverable-status
+      outputs:
+        - name: url
+          preview: '"three"'
```

Comment:
I wasn't really sure about the `"` around the string (as it must being `json`), so I gave a little try using `yaml` instead, and while it's indeed "nicer" for strings, for anything that's an object, `json` indeed looks much better imo

e.g.:

```
  - inputs:
    - name: image-builder
    name: deployer
    outputs:
    - digest: sha256:d6921e73a5877a89cd5c9e4c152a4c23f65bdda57507fd3a6881e7c9a78e6743
      lastTransitionTime: "2022-03-02T21:27:01Z"
      name: config
      preview: '{"deploy":[{"kapp":{}}],"fetch":[{"inline":{"paths":{"kapp-config.yml":"
```

vs

```
  - inputs:
    - name: image-builder
    name: deployer
    outputs:
    - digest: sha256:a97722bdd4787940f98dacd2434dcfc624ae8924bc6ffc960146d2e392f1650b
      lastTransitionTime: "2022-03-02T21:25:37Z"
      name: config
      preview: "deploy:\n- kapp: {}\nfetch:\n- inline:\n    paths:\n      kapp-config.yml:
```